tests/test_config.py

```diff
@@ -106,6 +108,20 @@ def test_init_studies(self):
         self.assertIn('sort_on_field', enrich_onion_github_params)
         self.assertIn('no_incremental', enrich_onion_github_params)
 
+        self.assertIn('gerrit', top_sections)
+        self.assertIn('enrich_demography:gerrit', top_sections)
+        self.assertIn('date_field', enrich_demography_gerrit_params)
+        self.assertIn('author_field', enrich_demography_gerrit_params)
+
+        self.assertIn('enrich_onion:gerrit', top_sections)
+        self.assertIn('in_index', enrich_onion_gerrit_params)
+        self.assertIn('out_index', enrich_onion_gerrit_params)
+        self.assertIn('data_source', enrich_onion_gerrit_params)
+        self.assertIn('contribs_field', enrich_onion_gerrit_params)
+        self.assertIn('timeframe_field', enrich_onion_gerrit_params)
+        self.assertIn('sort_on_field', enrich_onion_gerrit_params)
+        self.assertIn('no_incremental', enrich_onion_gerrit_params)
```

Comment:
I miss a check for 'seconds'. Shouldn't it be here?

Reply:
good catch